src/librustc_driver/lib.rs

```diff
@@ -347,6 +344,35 @@ pub trait CompilerCalls<'a> {
 #[derive(Copy, Clone)]
 pub struct RustcDefaultCalls;
 
+/**
+ * TODO remove these and use winapi 0.3 instead
+ *
+ * These are duplicated in
+ *   - bootstrap/compile.rs#L478
+ *   - librustc_errors/emitter.rs#L1253
+ */
```

Comment:
Could you use a line comments here? Also, you'll need to remove isatty from Cargo.lock/Cargo.toml.